docs/api.rst

```diff
@@ -52,6 +52,9 @@ Unitary effects that can be applied to one or more qubits.
     cirq.XX
     cirq.YY
     cirq.ZZ
+    cirq.rx
+    cirq.ry
+    cirq.rz
```

Comment:
You forgot to move `cirq.Rx` and friends into the deprecated section

Reply:
Thanks for catching! #2585.